### PR TITLE
Yup funding input

### PR DESCRIPTION
**Pull request by anissa-agahchen**
Screen Shot 2021-03-31 at 1 56 13 PM
# Overview

## Links to jira tickets
Set a limit on funding amount to match what the DB expects.

### CODE REVIEW

**Review by sdevalapurkar — APPROVED** [review]
I think you're adding a code smell here.... 😉

**Comment by anissa-agahchen** [comment]
> I think you're adding a code smell here.... 😉

File: app/src/features/projects/components/ProjectFundingItemForm.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const ProjectFundingFormArrayItemYupSchema = yup.object().shape({
     .transform((value) => (isNaN(value) && null) || value)
     .typeError('Must be a number')
     .min(0, 'Must be a positive number')
+    .max(9999999999)
```
Comment (sdevalapurkar) [inline app/src/features/projects/components/ProjectFundingItemForm.tsx]: this makes the error message that shows up say:
```
funding_amount must be less than.....
```

would it somehow be possible to change the message to say:
```
Funding amount must be less than.....
```

maybe by doing:
```
.max(9999999999, 'Funding amount must be less than...')
```

**Review by sdevalapurkar — CHANGES_REQUESTED** [review]

File: app/src/features/projects/components/ProjectFundingItemForm.tsx (hunk 1 above)
Comment (NickPhura) [reply app/src/features/projects/components/ProjectFundingItemForm.tsx]: "Funding amount must be less than $9,999,999,999"

Comment (anissa-agahchen) [reply app/src/features/projects/components/ProjectFundingItemForm.tsx]: updated

**Review by sdevalapurkar — APPROVED** [review]

**Pull request merged**